www/horj18/staticky_web/index.css

```diff
@@ -0,0 +1,316 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i');
```

Comment:
fonty se importuji v HTML head tagu pomoci <link>, takto je to spatne :)

navic, opravdu potrebujes vsechny ty podmnoziny fontu?